compute/rn/release-information/release-notes-22-12-update1.adoc

```diff
@@ -0,0 +1,93 @@
+:toc: macro
+== 22.12 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.12.xxx
+
+|Codename
+|Lagrange, 22.12 Update 1
+|Release date
+|Jan 9, 2023
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|TBD
+|===
+
+//Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+
+// LINK
+
+toc::[]
+
+=== New Features in Core
+// CWP-42365
+==== Filter Defender /api/vVERSION/defenders by TAS Foundation ID
+Introduces a new value,*tasFoundations*, in the *fields* query parameter to filter Defenders by TAS Foundation ID.
```

Comment:
@divyabhushan maybe it is worth mentioning that the filter is under Manage > Defenders > Deployed Defenders